docs/FAQ.md

```diff
@@ -377,18 +378,31 @@ for extensions.
 
 Theia doesn't allow you to reuse your existing VS Code config.
 
+## What's the difference between code-server and OpenVSCode-Server?
+
+code-server and OpenVSCode-Servr both allow you to access VS Code via a
+browser. The two projects also use their own [forks of VS Code](https://github.com/cdr/vscode) to
+leverage modern VS Code APIs and stay up to date with the upsteam version.
```

Comment:
@TeffenEllis - you have much more context here, and it very is possible I am not understanding this correctly. would love your 👀

Reply:
This looks good to me (in general!)